api/institutions/views.py

```diff
@@ -521,3 +528,52 @@ def get_default_queryset(self):
         institution = self.get_institution()
         search = UserInstitutionProjectCounts.get_current_user_metrics(institution)
         return self._make_elasticsearch_results_filterable(search, id=institution._id, department=DEFAULT_ES_NULL_VALUE)
+
+
+class _NewInstitutionUserMetricsList(InstitutionMixin, ElasticsearchListView):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        IsInstitutionalMetricsUser,
+    )
+
+    required_read_scopes = [CoreScopes.INSTITUTION_METRICS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+
+    view_category = 'institutions'
+    view_name = 'institution-user-metrics'
+
+    serializer_class = NewInstitutionUserMetricsSerializer
+
+    default_ordering = '-storage_byte_count'
+    ordering_fields = frozenset((
+        'user_name',
+        'department',
+        'month_last_login',
+        'account_creation_date',
+        'public_projects',
+        'private_projects',
+        'public_registration_count',
+        'embargoed_registration_count',
+        'published_preprint_count',
+        'public_file_count',
+        'storage_byte_count',
+    ))
+
+    def get_default_search(self):
+        _yearmonth = InstitutionalUserReport.most_recent_yearmonth()
+        if _yearmonth is None:
+            return None
+        return (
+            InstitutionalUserReport.search()
+            .filter('term', report_yearmonth=str(_yearmonth))
+            .filter('term', institution_id=self.get_institution()._id)
+        )
+
+
+institution_user_metrics_list_view = view_toggled_by_feature_flag(
```

Comment:
@futa-ikeda Do we have a clean-up ticket for after the flag is flipped? This is fine for feature flipping, but is a weird import side-effect in the long term.

Reply:
added a cleanup ticket: https://openscience.atlassian.net/browse/ENG-6223